packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -0,0 +1,244 @@
+    };
+
+    const updateErrors = (index, item_index, sub_index) => {
+        let error_count = 0;
+        const errors = {};
+        errors.data = [...document_upload_errors];
+        if (typeof errors.data[index] === 'object') {
+            delete errors.data[index][item_index].files[sub_index];
+            const has_other_errors = errors?.data[index]?.[item_index]?.files?.some(error => error !== null);
+            if (!has_other_errors) {
+                delete errors?.data[index]?.[item_index];
+            }
+            document_upload_errors.forEach(e => {
+                error_count += Object.keys(e).length;
+            });
+            if (error_count === 0) {
+                errors.data = [];
+            }
+        }
+        setDocumentUploadErrors(errors?.data);
+        setIsSubmitButtonDisabled(true);
+    };
+    const handleSubmit = () => {
+        try {
+            setFormState({ ...form_state, ...{ is_btn_loading: true } });
+            const { data: formValues } = form_ref.current.values;
+            const uploader = new DocumentUploader({ connection: WS.getSocket() });
+            if (form_ref.current.errors.length > 0) {
+                // Only upload if no errors and a file has been attached
+                return;
+            }
+            Object.keys(formValues).forEach(card_key => {
+                Object.keys(formValues[card_key]).forEach(async card_item_key => {
+                    const payment_method_details = formValues[card_key][card_item_key];
+                    if (payment_method_details.files.length > 0) {
+                        const processed_files = await readFiles(payment_method_details.files, fileReadErrorMessage, {
+                            documentType: DOCUMENT_TYPE.proof_of_ownership,
+                            proof_of_ownership: {
+                                details: {
+                                    email: client_email,
+                                    payment_identifier: payment_method_details.payment_method_identifier,
+                                },
+                                id: payment_method_details.id,
+                            },
+                        });
+                        processed_files.forEach(async (processed_file, sub_index) => {
+                            const response = await uploader.upload(processed_file);
+                            if (response.warning) {
+                                if (response.warning.trim() === 'DuplicateUpload') {
+                                    form_ref.current.errors.data = document_upload_errors;
+                                    const { data: form_errors } = form_ref?.current?.errors;
+                                    if (typeof form_errors[card_key] !== 'object') {
+                                        form_errors[card_key] = {};
+                                    }
+                                    form_errors[card_key][card_item_key] =
```

Comment:
and here also similar question about assigning the values :)

Reply:
This is similar to the above, but this is to do with errors after submitting the document